server/controllers/admin/projects.js

```diff
@@ -117,19 +116,23 @@ exports.create = function create(req, res, next) {
  */
 exports.update = async function update(req, res, next) {
```

Comment:
As a general comment on the design of BHIMA: when the user updates information about a project they are actively using, they must sign out to refresh the session information.  To avoid this, it would be nice to call the `loadSessionInforation()` from `auth` here ([link](https://github.com/Third-Culture-Software/bhima/blob/master/server/controllers/auth.js#L130)) if the project id of the user session is the same as the project id being updated.

You can choose to tackle this in this PR or we can put this off for a later PR.

Reply:
I agree with this concern.   I suggest adding the `loadSessionInformation()` before merging this PR.
This is especially true you plan to switch between projects regularly.